vulcan/hebe/data/_attendance.py

```diff
@@ -0,0 +1,150 @@
+    :var int ~.id: aux presence ID
+    :var str ~.justification_status: attendance justification status
+    :var `~vulcan.hebe.data.PresenceType` ~.presence_type: presence type
+    :var str ~.note: attendance note
+    :var str ~.public_resources: attendance public esources
+    :var str ~.remote_resources: attendance remote resources
+    :var `~vulcan.hebe.model.TeamVirtual` ~.group: group, that has the lesson
+    :var bool ~.visible: attendance visibility
+
+
+    """
+
+    lesson_id: int = IntegerField(key="LessonId")
+    id: int = IntegerField(key="Id")
+    lesson_number: int = IntegerField(key="LessonNumber")
+    global_key: str = StringField(key="GlobalKey")
+    lesson_class_id: int = IntegerField(key="LessonClassId")
+    lesson_class_global_key: str = StringField(key="LessonClassGlobalKey")
+    calculate_presence: bool = BooleanField(key="CalculatePresence")
+    replacement: bool = BooleanField(key="Replacement")
+    subject: Subject = ChildField(Subject, key="Subject", required=False)
+    topic: str = StringField(key="Topic", required=False)
+    teacher: Teacher = ChildField(Teacher, key="TeacherPrimary", required=False)
+    second_teacher: Teacher = ChildField(
+        Teacher, key="TeacherSecondary", required=False
+    )
+    main_teacher: Teacher = ChildField(Teacher, key="TeacherMod", required=False)
+    team_class: TeamClass = ChildField(TeamClass, key="Clazz", required=False)
+    class_alias: str = StringField(key="GroupDefinition", required=False)
+    date: DateTime = ChildField(DateTime, key="Day", required=False)
+    time: TimeSlot = ChildField(TimeSlot, key="TimeSlot", required=False)
+    date_modified: DateTime = ChildField(DateTime, key="DateModify", required=False)
+    aux_presence_id: int = IntegerField(key="AuxPresenceId", required=False)
+    justification_status: str = StringField(key="JustificationStatus", required=False)
+    presence_type: PresenceType = ChildField(
+        PresenceType, key="PresenceType", required=False
+    )
+    note: str = StringField(key="Note", required=False)
+    public_resources: str = StringField(key="PublicResources", required=False)
+    remote_resources: str = StringField(key="RemoteResources", required=False)
+    group: TeamVirtual = ChildField(TeamVirtual, key="Distribution", required=False)
+    visible = BooleanField(key="Visible", required=False)
+
+    @classmethod
+    async def get(
+        cls, api, last_sync, deleted, date_from, date_to, **kwargs
+    ) -> Union[AsyncIterator["Attendance"], List[int]]:
+        if date_from == None:
+            date_from = datetime.date.today()
+        if date_to == None:
+            date_to = date_from
+        date_to = date_to + datetime.timedelta(
+            days=1
+        )  # Vulcan requires the date_to to be one greater the date it is supposed to be
+        """
+        :rtype: Union[AsyncIterator[:class:`~vulcan.hebe.data.Attendance`], List[int]]
+        """
```

Comment:
Do not place docstrings after any code, they should be at the beginning of the function.